src/styles/AddImage.css.js

```diff
@@ -0,0 +1,14 @@
+import { css } from 'react-emotion';
```

Comment:
💅

Reply:
This is a cool library. Very similar to StyledComponents, and I like it. 👩‍🎤